ecosystem/typescript/sdk/src/indexer/queries/getAccountCoinsData.graphql

```diff
@@ -0,0 +1,11 @@
+query getAccountCoinsData($owner_address: String, $offset: Int, $limit: Int) {
```

Comment:
noob q: are these graphql queries used anywhere? why we need to have these queries if they are already exported from `queries.ts`?

Reply:
`queries.ts` is generated based on the queries in `.graphql` files. so write a gql query in a `.graphql` file, then generate those queries into a `.ts` file and use it.